README.md

```diff
@@ -25,10 +25,6 @@ Homebrew is highly recommended for PHP:
    * Run `npm install` to install Node.js packages.
    * Run `composer install` to install PHP packages.
    * If you have not already installed `node-lambda` as a global package, run `npm install -g node-lambda`.
```

Comment:
remove "If you have not already installed `node-lambda`.."

Reply:
Fixed